### ISSUE TITLE
Custom salvage update to 1.8

### ISSUE THREAD

**Issue by st3fan21** [BattletechModders/CustomSalvage#3]
Hey man,

Your mod CUSTOM SALVAGE is awesome, with it there is no need to search for all possbile versions of a mech, you can just create different versions with at least some parts.

Are u going to update to 1.8?

**Comment by sqparadox** [comment]
This worked in 1.8, hasn't been tested in 1.8.1:
[CustomSalvage.zip](https://github.com/BattletechModders/CustomSalvage/files/3953021/CustomSalvage.zip)

That's the mod and source code, 1 file needed an include modified and a single other line edit.

I updated a bunch of mods for myself when 1.8 came out. I don't have time to be supporting them right now, which is why I haven't forked or uploaded anything.

**Comment by st3fan21** [comment]
Yes seems to be working, but i also updated mech engineer and custom components to 1.8.1 and now the game does not recognize my saved games anymore :(

**Comment by st3fan21** [comment]
I tested everything and it s when i install mech engineer 1.8.1 that the game no longer recognizes my old saves
